src/routes/user.tsx

```diff
@@ -170,7 +170,26 @@ const DeleteUserButton = ({ user }: { user: UserInfo }) => {
   )
 }
 
-const SetPasswordButton = ({ user }: { user: UserInfo }) => {
+const alertOnSaveAdmin = async (selfEdit: boolean, user: UserInfo) => {
```

Comment:
Can we rename `selfEdit` to `isCurrentUser` for consistency? (I think they mean the same thing but it is easier to understand if they match)